user_docs/en/userGuide.t2t

```diff
@@ -2925,6 +2926,30 @@ If you install an add-on with paid components and change your mind about using i
 The Add-on Store is accessed from the Tools submenu of the NVDA menu.
 To access the Add-on Store from anywhere, assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+++ Note on security and privacy when using Add-ons ++[AddonSecurityAndPrivacy]
+Installing add-ons in NVDA leads to integration of external code into NVDA's functionality in order to enhance NVDA or make new features possible.
+Add-ons can also use external libraries and third party services to serve the purpose and provide the features for which they have been developed. They can be developed by every person or company.
+
+Automated code checks for add-ons happen when they are submitted to NVDA’s Add-on Store. This means e.g.  they will undergo  a basic automated check for vulnerabilities by security analysis tools such as codeQL.
+There isn't currently any formal review process for add-ons performed by a human.
+However, many add-ons have discussions areas where users can exchange feedback. The [community review area #AddonStoreReviews] can be accessed via the actions menu of the add-on.
+
+Installed Add-ons or extensions (not only in NVDA) might in general introduce security and/or privacy vulnerabilities, depending on the permissions they need and actions they perform in order to provide the desired functionality.
+Risks can be e.g.
+- Insecure network connections
+- Files stored with insecure file permissions or in an unprotected location
+- Sensitive information written to an easily available log file
+- Vulnerabilities in third-party libraries
+- Cryptographic vulnerabilities, and more.
+-
+
+Users install NVDA add-ons at their own risk. Therefore, everyone should be aware of following aspects when installing them:
+- Check out the developer’s website to see if it’s a serious source you can trust.
+- Read the description carefully. Does the add-on need questionable permissions? Does it track data? Does it share sensitive data with other sources that you don’t trust?
+- Check out the [community reviews #AddonStoreReviews] for the add-on. Are there any complaints about the add-on? Are there any reports about data being taken, or for anything that makes you feel unsafe?
+- The risk of vulnerabilities increases the more add-ons you installed. So be careful to keep the overview of the sources your add-ons come from.

```

Comment:
Here is some improved wording.
That said, I'm not sure that the second concept logically follows from the first.

```suggestion
- The risks of vulnerabilities increase the more add-ons you install. So be careful to keep an overview of the sources your add-ons come from.
```

Reply:
I don't think changing the statement in this way is what I meant. Add-ons in the add-on store can be risky even after automated checks as well because they don't have any human review. So questionable actions performed by an add-on cannot be detected by any statistical security analysis tool. It is still the responsibility of the user currently to deduct from the add-on description and the source of the add-on whether a certain feature is really needed in NVDA or whether it introduces too much vulnerabilities for the subjective feeling of the user.